analyses/molecular-subtyping-NBL/01-Subtyping.Rmd

```diff
@@ -0,0 +1,315 @@
+---
+title: "01-subtyping"
+author: "Aditya Lahiri, Eric Wafula, Jo Lynne Rokita"
+date: "11/14/2022"
+output: html_notebook
+---
+# Objective
+To molecularly subtype neuroblastoma, ganglioneuroblastoma, and ganglioneuroma samples into MYCN 
+amplified or MYCN non-amplified. This script loads the table `Alteration_Table.tsv` from the 
```

Comment:
Question for @afarrel and JMM - do we want to separate the subtypes as "NBL, ganglioNB, and ganglioneuroma (some abbreviations)" or just leave all as "NBL, subtype"?